third_party/composer/autoload_psr4.php

```diff
@@ -6,9 +6,43 @@
 $baseDir = dirname($vendorDir);
 
 return array(
-    'Twig\\' => array($vendorDir . '/twig/twig/src', $vendorDir . '/twig/twig/src'),
```

Comment:
Should this be tracked?

Reply:
ping @olleharstedt

Reply:
This file should be whatever "composer install --no-dev" generates.